molgenis-core-ui/src/main/resources/js/thememanager.js

```diff
@@ -1,109 +1,56 @@
-			event.preventDefault();
-			updatePageTheme();
-			$.ajax({
-				contentType : 'application/json',
-				type : 'POST',
-				url : molgenis.getContextUrl() + '/set-bootstrap-theme',
-				data : '"' + styleName + '"',
-                success: function () {
-					molgenis.createAlert([ {
-						'message' : 'Succesfully updated the application theme'
-					} ], 'success');
-				}
-			});
-		});
-
-        $('#show-add-theme-btn').on('click', function () {
-            $('#bootstrap-theme-select').prop('disabled', true);
-            $('#save-selected-bootstrap-theme').prop('disabled', true);
-            $('#show-add-theme-btn').prop('disabled', true);
-            $('#add-theme-container').show();
-            toggleAddBtn();
-        });
-
-        $('#cancel-add-themes-btn').on('click', function () {
-            $('#bootstrap-theme-select').prop('disabled', false);
-            $('#save-selected-bootstrap-theme').prop('disabled', false);
-            $('#show-add-theme-btn').prop('disabled', false);
-            $('#add-theme-container').hide();
-        });
-
-        $('#add-themes-btn').on('click', function () {
-            var bs3ThemeFile = $('#bootstrap3-file')[0].files[0];
-            var bs4ThemeFile = $('#bootstrap4-file')[0].files[0];
-
-            var data = new FormData();
-            data.append('bootstrap3-style', bs3ThemeFile);
-            if (bs4ThemeFile) {
-                data.append('bootstrap4-style', bs4ThemeFile);
-            }
-
-            $.ajax({
-                url: molgenis.getContextUrl() + '/add-bootstrap-theme',
-                data: data,
-                cache: false,
-                contentType: false,
-                processData: false,
-                type: 'POST',
-                success: function (theme) {
-
-                    molgenis.createAlert([{
-                        'message': 'Successfully added the application theme'
-                    }], 'success');
-
-                    $('#save-selected-bootstrap-theme').prop('disabled', false);
-                    $('#show-add-theme-btn').prop('disabled', false);
-                    $('#add-theme-container').hide();
-
-                    var newThemeOption = $("<option/>").val(theme.location).text(theme.name);
-
-                    $('#bootstrap-theme-select')
-                        .append(newThemeOption)
-                        .val(theme.location)
-                        .prop('disabled', false);
-
-                    updatePageTheme();
-                },
-                error: function (response) {
-
-                    var detailMessage = response.responseJSON ? " :" + response.responseJSON.errors[0].message : ".";
-
-                    molgenis.createAlert([{
-                        'message': 'Error adding theme' + detailMessage
-                    }], 'error');
-                }
-            });
-
-
+//TODO: https://unpkg.com/@molgenis/molgenis-theme/css/
```

Comment:
Shall we make the URL an app setting?

Reply:
I would rather do this via the proxy. Can we add a custom unpkg endpoint to the config instead?

Reply:
Discussed it also with @connoratrug. The @molgenis/molgenis-theme package was renamed to @molgenis-ui/molgenis-theme to make use of the existing unpkg endpoints. A custom endpoint in molgenis-theme dev can be used to serve local files.
![Screenshot_20201028_155323](https://user-images.githubusercontent.com/48799939/97453184-c8ef0200-1935-11eb-8407-78431b2335ef.png)
![Screenshot_20201028_155336](https://user-images.githubusercontent.com/48799939/97453185-c9879880-1935-11eb-96f1-eef262c08b14.png)